doc/admin-guide/plugins/lua.en.rst

```diff
@@ -1838,6 +1838,7 @@ Socket address family
 
     TS_LUA_AF_INET (2)
     TS_LUA_AF_INET6 (10)
+    TS_LUA_AF_UNIX (1)
```

Comment:
I think these variables are really used when we are making changes to the outgoing address. So there is no need to support AF_UNIX here